docs/CHEATCODES.md

```
@@ -45,6 +45,17 @@ Arguments with stars `(argument*)` can be omitted.
     - `slaughtersong` - triggers the Starship event.
 - `inquisarti` - triggers the Artifact Loan quest.
 - `govmission` - spawns governor missions on all planets.
+- `inquismission (mission*)` - triggers an inquisition mission. You must have met the inquisition for this to work
+    - `planet` - investigate planet mission
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)